src/components/Dashboard/PurposeTable.tsx

```diff
@@ -0,0 +1,90 @@
+import React, {useEffect} from "react";
+import {getAllProcesses} from "../../api";
+import {Dpia, Process} from "../../constants";
+import {RouteComponentProps} from "react-router-dom";
+import {HeadingLarge} from "baseui/typography";
+import {Spinner} from "baseui/spinner";
+import {Cell, HeadCell, Row, Table} from "../common/Table";
+import {useTable} from "../../util/hooks";
+import {StyleObject} from "styletron-standard";
+import {codelist, ListName} from "../../service/Codelist";
+import RouteLink from "../common/RouteLink";
+import {intl} from "../../util";
+
+interface PathProps {
+  filterName?: 'PVK',
+  filterValue?: string
+}
+
+const cellStyle: StyleObject = {
+  maxWidth: '25%',
+  wordBreak: "break-word"
+}
+
+const PurposeTable = (props: RouteComponentProps<PathProps>) => {
```

Comment:
Ny tabell? Har vi diskutert hvordan ting skal se ut?

Reply:
Jeg snakket med @MugunthanD om det.

Reply:
Jeg får bare tallene for behandlinger fra dashboard API. Men vi trenger en tabel å vise dem.